### DISCUSSION TITLE
Help me plsplspls

### DISCUSSION THREAD

**Discussion by influence412** [chr233/ASFEnhance#197] [提问求助 / Support]
How to craft a steam community badge of game from level 0 to 5?
"Craftbadge" command only crafts up to level 3

**Comment by chr233** [comment]
this command craft every craftable badge once at one time, if u want to creft 0 to 5 levels, use this command five times

**Reply by influence412** [reply]
The command only works the first 3 uses, then it does not create an badge
![image](https://github.com/chr233/ASFEnhance/assets/154758173/4664199f-1632-4c4b-b434-728819579302)

**Reply by chr233** [reply]
do you got enough cards to craft?

**Reply by influence412** [reply]
Sure

**Reply by influence412** [reply]
first use of command craft badge from 0lvl to 1lvl
second use craft badge from 1lvl to 3lvl
And next uses dont craft.

**Reply by influence412** [reply]
![image](https://github.com/chr233/ASFEnhance/assets/154758173/a21bf744-7349-4207-82a6-1cfef3465535)

**Reply by chr233** [reply]
can u give me more information about what game's badge and what's your current level of the badge?

**Reply by influence412** [reply]
It is necessary? I'm out of cards. I’ll explain from start to finish: with five sets of game badge cards, the command crafts with the first use from level 0 to level 1, the second use crafts from level 1 to level 3, and next uses do nothing.
![image](https://github.com/chr233/ASFEnhance/assets/154758173/c0d78a5e-512a-47b9-a38b-9d85323314a6)
I encountered an identical error back in November, now I started crafting again and encountered it again

**Reply by chr233** [reply]
I think there was a same issue, I will review code when I have time.

**Reply by influence412** [reply]
pls fix it

**Reply by chr233** [reply]
can u give me your profile link

**Reply by chr233** [reply]
![image](https://github.com/chr233/ASFEnhance/assets/13798576/736c1e9a-2576-48e5-bede-a51cca252e33)
I got the same issue

**Comment by chr233** [comment]
fixed in 2.0.9.3
